### DISCUSSION TITLE
32讲的水龙头应用的一个问题

### DISCUSSION THREAD

**Discussion by RealWillsLi** [AmazingAng/WTF-Solidity#456] [Q&A] [answered]
![OZTEQII1QL575X~54GLF(ZB](https://user-images.githubusercontent.com/89158403/226892479-0e1fa6ba-60f7-499a-bbf2-5d1be5b157ca.png)
用IERC20和ERC20创建的区别是什么

**Comment by AmazingAng** [comment, marked as answer]
IERC20是ERC20的接口合约，这里是用于调用目标ERC20代币合约的。
可以再看一下接口那一讲。

**Reply by RealWillsLi** [reply]
好，多谢A哥
